src/js/utils/ajax.js

```diff
@@ -150,7 +150,7 @@ function _requestError(message, options) {
 }
 
 function _error(options, message, code, error) {
-    options.onerror(message, options.url, options.xhr, new PlayerError(message, code, error));
+    options.onerror(message, options.url, options.xhr, new PlayerError(MSG_CANT_LOAD_PLAYER, code, error));
```

Comment:
We can pass a specific key depending on the scenario; i.e. 'File not found' is really a `cantPlayVideo `